analyzer/util.go

```diff
@@ -421,6 +421,10 @@ func containsReflectValue(t types.Type) bool {
 			return false
 		}
 		seen[t] = struct{}{}
+		// If the unaliased type is different, use that.
```

Comment:
Presumably we don't need to check underlying or the struct/array bits for an aliased type?

Reply:
Yes, we can replace t with u here, and u will have all the required properties like struct members that we needed to search for in t.